python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt_tests/dlt_tests/test_dlt.py

```diff
@@ -61,26 +43,27 @@ def example_pipeline_assets(
     )
     assert res.success
 
-    with duckdb.connect(database=EXAMPLE_PIPELINE_DUCKDB, read_only=True) as conn:
+    temporary_duckdb_path = dlt_pipeline.pipeline_name + ".duckdb"
```

Comment:
nit: just to make this explicit. There's also some other references that could use string formatting instead of string concatenation.

```suggestion
    temporary_duckdb_path = f"{dlt_pipeline.pipeline_name}.duckdb"
```